definitions/npm/openai_v4.x.x/flow_v0.142.x-/openai_v4.x.x.js

```diff
@@ -0,0 +1,2509 @@
+     * caste. Hateful content aimed at non-protected groups (e.g., chess players) is
+     * harrassment.
+     */
+    hate: boolean;
+
+    /**
+     * Hateful content that also includes violence or serious harm towards the targeted
+     * group based on race, gender, ethnicity, religion, nationality, sexual
+     * orientation, disability status, or caste.
+     */
+    // "hate/threatening": boolean;
+
+    /**
+     * Content that promotes, encourages, or depicts acts of self-harm, such as
+     * suicide, cutting, and eating disorders.
+     */
+    // "self-harm": boolean;
+
+    /**
+     * Content that encourages performing acts of self-harm, such as suicide, cutting,
+     * and eating disorders, or that gives instructions or advice on how to commit such
+     * acts.
+     */
+    // "self-harm/instructions": boolean;
+
+    /**
+     * Content where the speaker expresses that they are engaging or intend to engage
+     * in acts of self-harm, such as suicide, cutting, and eating disorders.
+     */
+    // "self-harm/intent": boolean;
+
+    /**
+     * Content meant to arouse sexual excitement, such as the description of sexual
+     * activity, or that promotes sexual services (excluding sex education and
+     * wellness).
+     */
+    sexual: boolean;
+
+    /**
+     * Sexual content that includes an individual who is under 18 years old.
+     */
+    // "sexual/minors": boolean;
+
+    /**
+     * Content that depicts death, violence, or physical injury.
+     */
+    violence: boolean;
+
+    /**
+     * Content that depicts death, violence, or physical injury in graphic detail.
+     */
+    // "violence/graphic": boolean;
+  }
+
+  /**
+   * A list of the categories along with their scores as predicted by model.
+   */
+  declare interface Moderation$CategoryScores {
+    /**
+     * The score for the category 'harassment'.
+     */
+    harassment: number;
+
+    /**
+     * The score for the category 'harassment/threatening'.
+     */
+    // "harassment/threatening": number;
+
+    /**
+     * The score for the category 'hate'.
+     */
+    hate: number;
+
+    /**
+     * The score for the category 'hate/threatening'.
+     */
+    // "hate/threatening": number;
+
+    /**
+     * The score for the category 'self-harm'.
+     */
+    // "self-harm": number;
+
+    /**
+     * The score for the category 'self-harm/instructions'.
+     */
+    // "self-harm/instructions": number;
+
+    /**
+     * The score for the category 'self-harm/intent'.
+     */
+    // "self-harm/intent": number;
+
+    /**
+     * The score for the category 'sexual'.
+     */
+    sexual: number;
+
+    /**
+     * The score for the category 'sexual/minors'.
+     */
+    // "sexual/minors": number;
+
+    /**
+     * The score for the category 'violence'.
+     */
+    violence: number;
+
+    /**
+     * The score for the category 'violence/graphic'.
+     */
+    // "violence/graphic": number;
```

Comment:
I had trouble with moderation field names with symbols because flow gives an "unsupported-syntax" error.

```
    Illegal name. [unsupported-syntax]
    
       2281|     "harassment/threatening": boolean;
                 ^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
These category names are fixed in 6e1ea59d.